dev/com.ibm.ws.opentracing/src/com/ibm/ws/opentracing/OpentracingClientFilter.java

```diff
@@ -167,7 +169,11 @@ public void filter(ClientRequestContext clientRequestContext,
         }
 
         if (continuation == null) {
-            Tr.error(tc, "OPENTRACING_NO_SPAN_FOR_RESPONSE_TO_OUTBOUND_REQUEST");
```

Comment:
Given that this message is no longer used in code, it should be removed from NLS file